unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/LoadingHUD/Bridge/LoadingBridge.cs

```diff
@@ -20,5 +22,11 @@ public void SetLoadingScreen(string jsonMessage)
         if (!string.IsNullOrEmpty(payload.message))
             DataStore.i.HUDs.loadingHUD.message.Set(payload.message);
         DataStore.i.HUDs.loadingHUD.showTips.Set(payload.showTips);
+
+        if (!prewarmedShaderVariants)
+        {
+            Resources.Load<ShaderVariantCollection>("ShaderVariantCollections/shaderVariants-selected").WarmUp();
```

Comment:
whats is the initiative of moving the prewarm here? i find it less cohesive, since this class is handling the loading screen behaviour but now also has a shader prewarm which makes it harder to find

Reply:
The initiative is that the landing gets completely frozen for around 5 seconds when the user is interacting with the login options.

Maybe we can do this event based and subscribe from `Main`?